geode-deployment/build.gradle

```diff
@@ -13,4 +13,4 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
- */
```

Comment:
This appears to be an empty project that builds nothing. Shouldn't it pull in our `${rootDir}/gradle/java.gradle` project configuration or something?

Reply:
This module is really just a logical abstraction to keep all the deployment related projects together. I don't think there is anything in this project that we need to build, etc... If you believe that we need to add some extra behavior here, please let me us know, but this project is essentially a contextual module rather than a module that provides parent behavior.

Reply:
If it serves no purpose other than directory organization, then drop the `build.gradle` file, and remove its entry from `rootDir/settings.gradle`.